tools/ci/download-xs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+
+# Download HDF5 data
+if [[ ! -e $HOME/nndc_hdf5/cross_sections.xml ]]; then
```

Comment:
is there a particular reason you are looking at a specific file as opposed to presence of the entire directory? Why not simply:
```
if [[ ! -d $HOME/nndc_hdf5/ ]]; then
```

Reply:
Travis CI will create this directory because it's indicated as a cached directory. Thus, it's not sufficient to just check for the existence of the directory.